tensorrt_llm/_torch/attention_backend/sparse/minimax_m3/msa_backend.py

```
@@ -293,6 +309,54 @@ def msa_decode_dense_plan(self) -> Optional[tuple]:
+        The eager GQA and dense plans must pass use_fp8_kvcache so the inline
+        sparse-prefill path selects the FP8 AOT kernels; it is a no-op for the
+        decode planner. Mirrors the k_paged.dtype check in run_msa_paged_gqa.
+        """
+        kv_cache_manager = self.kv_cache_manager
+        if kv_cache_manager is None:
+            return False
+        try:
+            buffers = kv_cache_manager.get_buffers(0, kv_layout="HND")
+        except TypeError:
+            buffers = kv_cache_manager.get_buffers(0)
+        except Exception:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)